gulpfile.js

```diff
@@ -1,26 +1,24 @@
-'use strict';
```

Comment:
I would be glad to accept a PR indeed. The only restriction is that it must run on both Unix and Windows.